template.json

```
@@ -31,12 +35,13 @@
       "stylelint-config-property-sort-order-smacss": "7.1.0"
     },
     "scripts": {
-      "lint": "eslint ./src --ext .ts,.tsx",
-      "lint:fix": "eslint ./src --ext .ts,.tsx --fix",
+      "lint": "eslint ./src ./cypress --ext .ts,.tsx",
+      "lint:fix": "eslint ./src ./cypress --ext .ts,.tsx --fix",
       "stylelint": "stylelint '**/*.scss'",
       "stylelint:fix": "stylelint '**/*.scss' --fix",
       "codebase:lint": "yarn lint && yarn stylelint",
-      "codebase:fix": "yarn lint:fix && yarn stylelint:fix"
+      "codebase:fix": "yarn lint:fix && yarn stylelint:fix",
+      "cypress:run": "cypress run"
```

we should add another command
```suggestion
      "cypress:run": "cypress run",
      "cypress:open": "cypress open"
```